spec/file_reference_installer/header_file_reference_installer_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+#TODO: Figure out shorter class name
+module Xcake
+  describe HeaderFileReferenceInstaller do
+    before :each do
+      @node = double("Node")
+    end
+
+    it "should not able to install folder" do
+      folder_path = 'Folder/'
+
+      allow(@node).to receive(:path).and_return(folder_path)
+      allow(File).to receive(:directory?).with(folder_path).and_return(true)
+
+      expect(HeaderFileReferenceInstaller.can_install_node(@node)).to be(false)
+    end
+
+    it "should not be able to install non header file" do
+      allow(@node).to receive(:path).and_return('File.txt')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.